test/2_test_fullpath_with_permit.ts

```diff
@@ -2216,13 +2216,16 @@ describe('Cashier and VoucherKernel', () => {
         await contractMockBosonRouter.deployed();
 
         //Set mock so that passing wrong payment type from requestCreateOrderETHETH to createPaymentMethod can be tested
+        await contractBosonRouter.pause();
         await contractVoucherKernel.setBosonRouterAddress(
           contractMockBosonRouter.address
         );
 
         await contractCashier.setBosonRouterAddress(
           contractMockBosonRouter.address
         );
+        await contractMockBosonRouter.pause();
+        await contractMockBosonRouter.unpause();
```

Comment:
Why does this test pause then immediately unpause the mock boson router without doing anything inbetween? Add a comment if this isn't an oopsy.

Reply:
It's needed in the current implementation. I added a comment explaining why.